hw/bsp/pinetime/pkg.yml

```diff
@@ -0,0 +1,37 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+pkg.name: "hw/bsp/pinetime"
+pkg.type: bsp
+pkg.description: "BSP definition for the Pine64 PineTime smartwatch"
+pkg.author: "Apache Mynewt <dev@mynewt.apache.org>"
+pkg.homepage: "http://mynewt.apache.org/"
+pkg.keywords:
+    - nrf52
+    - Pine64
+    - PineTime
+
+pkg.cflags:
+    - '-DNRF52'
+    # TODO: Move to more specific variable, however this currently causes a build failure '-DNRF52832_XXAA'
```

Comment:
I'm wondering which would be the build failure here. It looks like NRF52832_XXAA would be the right value.

Reply:
I believe some library didn't build because it expected `NRF52`. I will look at it again and fix the actual problem ;-)

Reply:
So the first build problem I get is in `hw/mcu/nordic/nrf52xxx/include/nrfx_config.h`. This seems easily solvable, however there are more assumptions in the code for `NRF52` to be defined. Most of the bsp use `-DNRF52`, so I suggest to leave the TODO as-is and fix this later.

Reply:
Ok, nevermind, this is dealt with on nrfx, `NRF52` is ok.